README.md

```diff
@@ -119,14 +121,16 @@ tkn pipeline start operator-hosted-pipeline \
   --param env=prod \
   --param preflight_min_version=0.0.0 \
   --param ci_min_version=0.0.0 \
+  --param hydra_secret_name=hydra-credentials \
+  --param hydra_secret_username_key=username \
+  --param hydra_secret_password_key=password \
   --workspace name=repository,volumeClaimTemplateFile=templates/workspace-template-small.yml \
   --workspace name=results,volumeClaimTemplateFile=templates/workspace-template.yml \
   --workspace name=registry-credentials-all,volumeClaimTemplateFile=templates/workspace-template-small.yml \
   --workspace name=registry-credentials,secret=registry-dockerconfig-secret \
   --workspace name=pyxis-ssl-credentials,secret=operator-pipeline-api-certs \
   --workspace name=prow-kubeconfig,secret=prow-kubeconfig \
   --workspace name=preflight-decryption-key,secret=preflight-decryption-key \
```

Comment:
What about all these other workspaces from secrets? Are you planning to submit additional pull requests for the rest? They all seem like they can be eliminated from the list of required `tkn` args.

Reply:
Oh I thought the task only involved ones that we had to export. Yeah I'll work on a follow-up PR to take care of the rest, can we merge this first to minimize conflicts?

Reply:
Since this includes an update to the tkn command for the CI pipeline, it'll affect partners who pull from our `main` branch. That requires partner-facing documentation updates so it'd be ideal to try and merge all the CI pipeline changes together.

Reply:
Okay sure. The conflicts should be relatively easy to deal with anyway.

Reply:
I looked over the rest of the secret workspaces, and I don't think setting them as env var the same way would make sense. The remaining secrets are secret files, which gets passed to commands/scripts as file paths. But if we convert them to use `secretKeyRef`, then the env var will be saving content of the file, which we'll then have to echo into a file, which actually creates the security risk we were trying to avoid in the first place.

However, it looks like we could maybe mount the secret files as a volume instead? Should I pursue that? e.g. https://kubernetes.io/docs/concepts/configuration/secret/#use-case-pod-with-ssh-keys

Reply:
Yes, please investigate the volume mount option further.

Reply:
Using volumeMount seems to work fine. But just to be clear, this will not change anything with the security. It'll just be for getting rid of the workspaces. We're still referencing files (except from a volumeMount instead of a workspace) and some tasks still need to `cat` the files and thus cannot use `set -x`

Reply:
Ok that makes sense :+1:. Anything we can do to reduce the amount of required params is welcome.